projects/nx-verdaccio/src/plugin/targets/package.targets.ts

```diff
@@ -7,6 +7,16 @@ import { EXECUTOR_PACKAGE_NPM_INSTALL } from '../../executors/pkg-install/consta
 export const TARGET_PACKAGE_INSTALL = 'nxv-pkg-install';
 export const TARGET_PACKAGE_PUBLISH = 'nxv-pkg-publish';
 
+
+/**
+ * Determines if the given project is a publishable project.
+ * A project qualifies as a publishable if it's of type 'library' (early return if it's not)
+ * and, when options tag filters are provided, and they have match with any config tag.
+ *
+ * @param projectConfig
+ * @param options
+ * @returns `true` if the project is a publishable; otherwise, `false`.
```

Comment:
```suggestion
 * Determines if the given project is a `publishable` package.
 * A project qualifies as a `publishable` if it's of type 'library'.
 * If tag filters are provided only projects passing the filter will return true.
 *
 * @param projectConfig
 * @param options
 * @returns `true` if the project is a publishable; otherwise, `false`.
```

I removed the mentioned early return as  implementation details pollute the docs for users.